src/main/resources/checkstyle-config.xml

```diff
@@ -27,7 +27,11 @@
 <module name="Checker">
 
   <module name="TreeWalker">
-
+    <module name="UnusedImports"/>
+    <module name="AvoidStarImport">
+      <property name="allowClassImports" value="false"/>
```

Comment:
There is already 'AvoidStarImport' rule. Please change it to use defaults.

Reply:
Thanks for catching. Fixed.